validator/db/kv/migration.go

```diff
@@ -0,0 +1,29 @@
+package kv
+
+import (
+	"context"
+
+	bolt "go.etcd.io/bbolt"
+)
+
+var migrationCompleted = []byte("done")
+
+type migration func(*bolt.Tx) error
```

Comment:
```suggestion
type migration func(context.Context, *bolt.Tx) error
```

In case future migration script wants to use context?

Reply:
Until we need it...

![image](https://user-images.githubusercontent.com/7246818/102299010-1257d880-3f07-11eb-850f-baaa95f724a8.png)

Reply:
Fair lol